### ISSUE TITLE
[Feature Request] Set site title using Environment Variable

### ISSUE THREAD

**Issue by Kisaragi-ng** [mlan/docker-gitweb#2]
Hello, thank you for creating gitweb with docker, this really simplify a lot of process when I want to publish a simple read-only repository.

I have read readme.md but still can't found information regarding this one, would it be possible to set site title using docker environment?

with current docker image, by default I got this in my web browser:

![1](https://github.com/user-attachments/assets/7ef4edf6-ca16-48fc-acdd-68d556b09309)

and then after creating changes inside the container, I managed to set it to name that I want;

```diff
@@ /etc/nginx/conf.d/gitweb.conf

server {
  listen *:80;
-  server_name localhost;
+  server_name Mygitproject;
  access_log  off;

  location /gitweb.cgi {
    root /usr/share/gitweb/;
    include fastcgi_params;
```

i got it like this:
![2](https://github.com/user-attachments/assets/08bf98a8-f3b0-4e8b-ab74-a80c457254a1)

I'm not sure why the *mygitproject* doesn't show with uppercase, but at very least that's how i'd like to have it as my site title.

Unless I'm mistaken, there's no such feature documented yet in the current docker image build, so I'm hoping we can have something like this to define / setup the site title:

```diff
@@ mlan/docker-gitweb/blob/master/demo/docker-compose.yml

services:
  repo-gui:
    image: mlan/gitweb
    ports:
      - "${CNT_BIND-127.0.0.1:8080}:80"     # HTTP port
    environment:
      - PROJECTS_LIST=                      # Scan repos when list is empty
+     - SITE_TITLE="My Git Project"
    cap_add:
      - sys_ptrace                          # Only here to help testing
```

thank you in advance

**Comment by mlan** [comment]
Hello @Kisaragi-ng 

Many thanks for you interest in this project and your excellent suggestion. I will take a look at it.

**Comment by mlan** [comment]
Fixed in 41a9064
